### PR TITLE
Auto refresh

### PR DESCRIPTION
**Pull request by Lunat1q**
Added auto refresh Pokemons on the map.
Pokemons being handled (added/removed) every 10 seconds.

+ [x] Auto refresh feature
+ [x] Timezone in label fixed

### CODE REVIEW

**Comment by rezz3x** [comment]
Would be nice to bring back the -ar Argument to change the Seconds.

File: pogom/app.py [hunk 1]
```diff
@@ -26,6 +26,10 @@ def fullmap(self):
                                lng=config['ORIGINAL_LONGITUDE'],
                                gmaps_key=config['GMAPS_KEY'])
 
+<<<<<<< HEAD
```
Comment (calebthompson) [inline pogom/app.py]: merge conflict markers here.

**Comment by volksport** [comment]
I think you see them, but there are some merge conflicts in some of those files.

**Comment by mkody** [comment]
Yes, if you could please use the value from `args.auto-refresh` if the user uses the `-ar` or `--auto-refresh` argument.

**Comment by Lunat1q** [comment]
Found an conflict, merging now.

**Comment by Lunat1q** [comment]
ok, fixed.
About -ar parameter.
Autorefresh speed being controlled by JavaScript, not Python.

**Comment by Lunat1q** [comment]
Also looks like i fixed wrong timezone in Pokemon label.

**Comment by volksport** [comment]
You can use server variables in the template to change the JS...

**Comment by Lunat1q** [comment]
i'm not that string in Python, but anyway, i think it should be easy for skilled Python master :)
still, actually, i vote for UI slider, to change request frequency

**Comment by JZ6** [comment]
+1 from me

**Pull request merged**